debug/debug_windows.go

```diff
@@ -0,0 +1,87 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// +build windows
+
+package debug
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/cihub/seelog"
+)
+
+const (
+	// Maximum file size in MB
+	MAX_FILE_SIZE           float64 = 10
+	// Maximum number of files generated in log rotation
+	MAX_ROLLS               int     = 24
+	// Messages will be in the format "time=... level=... msg=..."
+	SEE_LOG_CONFIG_TEMPLATE string  = `
+<seelog type="asyncloop">
+	<outputs formatid="main">
+		<rollingfile filename="%s" type="size"
+			maxsize="%d" archivetype="none" maxrolls="%d" />
+	</outputs>
+	<formats>
+		<format id="main" format="time=%%Date/%%Time level=%%LEV msg=%%Msg%%n"/>
+	</formats>
+</seelog>`
+)
+
+var (
```

Comment:
i don't think you need these vars to be global. please consider moving to methods themselves.

Reply:
They need to be global because the SetLogFilePath will overwrite them.